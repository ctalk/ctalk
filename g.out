 D compile
 M include/builtins.h
 M include/searchdirs.h
?? .gitignore
?? build/.confgroup
?? build/.confname
?? build/.confuserhomedir
?? build/.gccdonthavenoformatoverflowopt
?? build/.gcchaveframeworkopt
?? build/.gcchavelongdouble
?? build/.gl_h_path
?? build/.glew_h_path
?? build/.glx_h_path
?? build/.includedir
?? build/.llvm
?? build/.x_h_path
?? build/builtin_names
?? build/conftest
?? build/make_builtins
?? build/make_builtins.dSYM/
?? config.h
?? config.log
?? config.status
?? ctpp/.deps/
?? g.out
?? include/osx_i386_builtins.h
?? include/osx_ppc_builtins.h
?? lib/.deps/
?? libctpp/.deps/
?? libtool
?? src/.deps/
?? src/ctcc
?? src/ctdb
