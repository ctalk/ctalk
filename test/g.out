 D ../compile
 M ../include/builtins.h
 M ../include/searchdirs.h
?? ../build/builtin_names
?? ../build/conftest
?? ../build/make_builtins
?? ../build/make_builtins.dSYM/
?? ../config.h
?? ../config.log
?? ../config.status
?? ../g.out
?? ../include/osx_i386_builtins.h
?? ../include/osx_ppc_builtins.h
?? ../libtool
?? ../src/ctcc
?? ../src/ctdb
?? g.out
